web/viewer.js

```diff
@@ -901,6 +901,11 @@ var PDFViewerApplication = {
                      store.get('zoom', self.pdfViewer.currentScale);
           var left = store.get('scrollLeft', '0');
           var top = store.get('scrollTop', '0');
+          var rotation = store.get('rotation', '0') | 0;
+
+          if (rotation !== self.pageRotation) {
```

Comment:
The code has changed here a bit. Instead of calling this here, let's pass the rotation as an argument into setInitialView